src/scripts.js

```diff
@@ -287,6 +287,20 @@ const displaySelectedTripToBook = (formData, destination) => {
 
 };
 
+const checkIfSignedIn = () => {
+  let params = new URLSearchParams(window.location.search);
+  let userID = params.get('username')?.split('')?.splice(8,5)?.join('');
+  if(userID) {
+    let traveler = travelerRepo.data.find(traveler => traveler.id == userID)
```

Comment:
Could you just set currentTraveler to the return value of .find instead of traveler?  It might dry up the function a little bit, and you wouldn't need to declare a new variable. This is a cool way of doing this!